.flowconfig

```diff
@@ -1,14 +1,7 @@
-# [version]
-# 0.126.1
-#
```

Comment:
Is it okay to drop this?

Reply:
I don't see what value it gives us. I don't use it in `render-gateway`, `checksync` or `ancesdir`.